README.md

```diff
@@ -1,9 +1,11 @@
 # filetree
-A FileTree for r/datacurator
+A filetree for r/datacurator
 
-Hello and Welcome to the [r/datacurator](https://reddit.com/r/datacurator) filetree repository.  
+Hello and welcome to the [r/datacurator](https://reddit.com/r/datacurator) filetree repository.  
 
-We aim to create a unified filetree for all kinds of data.
+We aim to create a unified filetree for all kinds of data, for making it easier to store, categorise and retrieve it.
+
+This filetree is primarily based on the categorisation of data into directories based on the form of the data involved. For example, if a file is a .mp3 or .wav file, it would then be sorted into the audio top-level section. However, within subsections, the data is usually categorised based on category. If the previously mentioned audio file was a piece of music, it would be sorted into the artists subsection. However, if it was an audiobook, it would be sorted into the books subsection.
```

Comment:
> if a file is a .mp3 or .wav file, it would then be sorted into the audio top-level section

That is not the impression one should get here. This repository is aimed to create multiple examples of how one can deploy a branching based filetree for his personal data collection in a curated manner. I've purposely left that one out of the main readme so far, because it wasn't clear what this filetree would become.

Reply:
Okay, thanks for clarifying, this could definitely be expanded with a later version when multiple filetrees are available but for now I'll leave it as is. 👍